messages.py

```diff
@@ -89,10 +87,11 @@ def message_handler(self, data):
             session.commit()
 
         mod_roles = ("Owner", "Staff", "Founder", "Global Mod", "Mod")
-        if not (data["user_roles"][0] in mod_roles or user.friend):
+        if not (data["user_roles"][0] in mod_roles or user.friend or
+                self.bot_name == data["user_name"]):
```

Comment:
Should `.lower()` `bot_name` and `data["user_name"]`, in case the user types `botusername` (or any non-matching capitalization) into the config.